src/lib/components/Modal.svelte

```diff
@@ -1,24 +1,39 @@
 <script lang="ts">
 	import { fade } from 'svelte/transition';
+
+	import { modal } from '$lib/stores/modal';
+	import Icon from '$lib/components/Icon.svelte';
 </script>
 
-<div class="container" transition:fade={{ duration: 300 }}>
-	<div class="PopIn">
-		<slot />
+{#if $modal}
+	<div class="container" transition:fade={{ duration: 300 }}>
```

Comment:
Can we use the `dialog` HTML element?

https://www.youtube.com/watch?v=TAB_v6yBXIE

https://www.youtube.com/watch?v=RO40Opyizeg